Chess.py

```diff
@@ -0,0 +1,27 @@
+import termcolor as tc
```

Comment:
@kayraBo  Can you move this file to your homework directory:
`software-engineering-2223/homework/12_V/16_Kayra/homework2`